cmd/command.go

```diff
@@ -17,11 +19,14 @@ func Run(c Command, asSubcommand bool) {
 	var (
 		printVersion bool
 		logLevel     string
+		noColor      bool
 	)
 
 	log.SetOutput(ioutil.Discard)
 	FlagVersion(&printVersion)
 	FlagLogLevel(&logLevel)
+	FlagNoColor(&noColor)
+	color.Disable(noColor)
```

Comment:
This need to happen after `flagSet.Parse`, `noColor` won't be set correctly at this point in the command execution.

Reply:
ah, I moved it right after my last smoke test. Good catch.